__tests__/account/AccountCreateTransaction.test.ts

```diff
@@ -19,51 +19,75 @@ describe('AccountCreateTransaction', () => {
         const bodybytes = "Cg4KCAjcyQcQ258JEgIYAxICGAMYgMLXLyICCHhaPwoiEiDgyOwnWKWHn/rCJqE8DFFreZ5y41FBoN2Cj5TTeYiktzD//////////384//////////9/SgUI0MjhAw==";
 
         const tx = transaction.toProto().toObject();
-        expect(tx).toStrictEqual({
-            body: undefined,
+        let expectedTx = {
+            body: undefined as undefined | Object,
```

Comment:
was this necessary?

Reply:
Unfortunately yes, if I remove `as undefined | Object` I get an error while assigning to that field later in the test.